apps/site/assets/css/_colors.scss

```diff
@@ -83,7 +83,6 @@ $hp-hero-gradient-end: rgba(28, 30, 35, .8);
 $hp-hero-gradient: linear-gradient(to bottom, $hp-hero-gradient-start, $hp-hero-gradient-end);
 
 // General
-$link-hover-color: $brand-primary-darkest;
```

Comment:
Not sure we want to remove this - it's picked up by Bootstrap's SCSS in `apps/site/assets/node_modules/bootstrap/scss/_reboot.scss` line 163. ;)

Reply:
Ouch, I didn't see that -- I will restore it.